src/test/java/org/fisco/bcos/channel/client/CallContractTest.java

```diff
@@ -0,0 +1,303 @@
+                        address,
+                        "getArray",
+                        new StaticArray2(
+                                typeMap(
+                                        Arrays.asList(
+                                                BigInteger.valueOf(-1), BigInteger.valueOf(2)),
+                                        Int16.class)),
+                        new DynamicArray(
+                                typeMap(
+                                        Arrays.asList(BigInteger.valueOf(2), BigInteger.valueOf(2)),
+                                        Uint16.class)));
+        List<Type> returnList4 =
+                callContract.decode(
+                        contractResult.getOutput(),
+                        new TypeReference<StaticArray2<Int16>>() {},
+                        new TypeReference<DynamicArray<Int16>>() {});
+        System.out.println(
+                "call getArray: "
+                        + callContract.convertList((List<Type>) returnList4.get(0).getValue())
+                        + ", "
+                        + callContract.convertList((List<Type>) returnList4.get(1).getValue()));
+
+        List<List<BigInteger>> dyadicArray = new ArrayList<List<BigInteger>>();
+        dyadicArray.add(Arrays.asList(BigInteger.valueOf(-1), BigInteger.valueOf(2)));
+        dyadicArray.add(Arrays.asList(BigInteger.valueOf(-1), BigInteger.valueOf(992)));
+        byte[] bytes = new byte[] {'a', 'b'};
+        contractResult =
+                callContract.call(
+                        address,
+                        "newTest",
+                        new StaticArray2(typeMap(dyadicArray, StaticArray2.class, Int256.class)),
+                        new DynamicBytes(bytes));
+        List<Type> returnList5 =
+                callContract.decode(
+                        contractResult.getOutput(),
+                        new TypeReference<StaticArray2<StaticArray2<Int256>>>() {},
+                        new TypeReference<DynamicBytes>() {});
+        System.out.println(
+                "call newTest: "
+                        + callContract.convertListList(
+                                (List<StaticArray<Int256>>) returnList5.get(0).getValue())
+                        + ", "
+                        + new String((byte[]) returnList5.get(1).getValue())
+                        + ", "
+                        + dyadicArray);
+    }
+
+    static class TransactionCallback extends TransactionSucCallback {
+        TransactionCallback() {
+            try {
+                semaphore.acquire(1);
+            } catch (InterruptedException e) {
+                Thread.currentThread().interrupt();
+            }
+        }
+
+        @Override
+        public void onResponse(TransactionReceipt receipt) {
+            this.receipt = receipt;
+            semaphore.release();
+        }
+
+        public TransactionReceipt receipt;
+        public Semaphore semaphore = new Semaphore(1, true);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/app/fisco/web3sdk/pullRequest?prid=4732356)